src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -188,15 +191,49 @@ object HouseholdActor {
     private var personAndActivityToCav: Map[(Id[Person], Activity), BeamVehicle] = Map()
     private var personAndActivityToLegs: Map[(Id[Person], Activity), List[BeamLeg]] = Map()
 
+    private val realDistribution: UniformRealDistribution = new UniformRealDistribution()
+    realDistribution.reseedRandomGenerator(beamScenario.beamConfig.matsim.modules.global.randomSeed)
+
     override def loggedReceive: Receive = {
 
       case TriggerWithId(InitializeTrigger(tick), triggerId) =>
+        val homeCoordFromPlans = household.members
+          .flatMap(per =>
+            per.getSelectedPlan.getPlanElements.asScala.flatMap {
+              case act: Activity if act.getType == "Home" => Some(act.getCoord)
+              case _                                      => None
+            }
+          )
+          .headOption
+          .getOrElse(fallbackHomeCoord)
+
         val vehiclesByCategory =
           vehicles.filter(_._2.beamVehicleType.automationLevel <= 3).groupBy(_._2.beamVehicleType.vehicleCategory)
-        val fleetManagers = vehiclesByCategory.map { case (category, vs) =>
+
+        //We should create a vehicle manager for cars and bikes for all households in case they are generated during the simulation
+
+        val vehiclesByAllCategories = List(Car, Bike)
```

Comment:
There is already a central place with all categories. This is a special case.

Reply:
I guess what I meant is that instead of putting `List(Car, Bike)` directly to store it as a variable for use anywhere else it is needed:

`val allVehicleCategories = List(Car, Bike)`